go/client/identify_ui.go

```diff
@@ -23,6 +23,30 @@ func NewIdentifyTrackUIProtocol(g *libkb.GlobalContext) rpc.Protocol {
 	return keybase1.IdentifyUiProtocol(&IdentifyUIServer{ui})
 }
 
+func cliIsLoggedIn(g *libkb.GlobalContext) bool {
+	configCli, err := GetConfigClient(g)
+	if err != nil {
+		return false
+	}
+
+	currentStatus, err := configCli.GetCurrentStatus(context.TODO(), 0)
+	if err != nil {
+		return false
+	}
+
+	return currentStatus.LoggedIn
+}
+
+// NewIdentifyMaybeTrackUIProtocol returns either IdentifyUiProtocol with
+// IdentifyTrackUI or IdentifyUI depending if user is logged in or not.
+func NewIdentifyMaybeTrackUIProtocol(g *libkb.GlobalContext) rpc.Protocol {
+	if !cliIsLoggedIn(g) {
```

Comment:
@patrickxb Would you mind taking a look? Using one protocol over another whether user is logged in sounds like a good idea but I don't know how to do this correctly. Maybe when the service hits us back we can make the decision which UI to pass the call to.